src/azure-cli/HISTORY.rst

```diff
@@ -7,6 +7,10 @@ Release History
 
 * Add support for importing/exporting feature flags
 
+**ARM**
+
+* Fix Recovery Services Vault tags cannot be updated
```

Comment:
Please specify the command you changed here.

Reply:
Done